plugins/pulp_rpm/plugins/importers/yum/config_validate.py

```diff
@@ -25,3 +46,69 @@ def validate(config):
             msg += failure_message + '\n'
         msg = msg.rstrip()  # remove the trailing \n
         return False, msg
+
+
+def _validate_allow_keys(repo, config, error_messages):
```

Comment:
Also since this method needs to know the unit count of the repo and another validation check further down does too I think getting the repo_unit_count in the `validate()` method itself and passing it into these validation check methods as an argument would be efficient.